COMMUNICATIONS.md

```diff
@@ -0,0 +1,83 @@
+# OpenSearch Dashboards Communication
+
+- [Overview](#overview)
+- [Slack](#slack)
+- [Forum](#forum)
+- [Developer Office Hours](#developer-office-hours)
+    - [What it is](#what-it-is)
+    - [When](#when)
+    - [How to sign up](#how-to-sign-up)
+    - [FAQ](#faq)
+
+## Overview
+
+The purpose of this document is to provide information regarding the communication channels for OpenSearch Dashboards. All communication is subject to the [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md). Please see [CONTRIBUTING](CONTRIBUTING.md) if you're interested in contributing to the project.
+
+## Slack
+
+The OpenSearch project has a public workspace on [Slack](https://opensearch.slack.com). See the [Getting Started guide]() for steps to register and setup the workspace.
+
+Once registered, check out these channels for discussion of OpenSearch Dashboards topics:
+
+- [#dashboards](https://opensearch.slack.com/archives/C01QENNTGUD)
+- [#dashboards-ux](https://opensearch.slack.com/archives/C05389T9LJC)
+
+## Forum
+
+Slack conversations are not searchable outside the workspace. For this reason we encourage using the [OpenSearch Dashboards category](https://forum.opensearch.org/c/opensearch-dashboards/57) of the forum for technical support discussions or summarizing findings for the rest of the community.
+
+## Developer Office Hours
+
+### What it is
+
+A recurring 1-hour virtual meeting for community developers to chat with [OpenSearch Dashboards project maintainers](MAINTAINERS.md). Priority will be given to topics that are signed-up in advance, but ad-hoc discussions are welcome in any remaining time.
+
+While we'll always prioritize asynchronous communication, sometimes a community call is the most effective and efficient venue to share information and knowledge. Some reasons to sign up:
+
+1. Review a proposal or technical design for a new feature in OpenSearch Dashboards or an OpenSearch Dashboards plugin
+2. Learn more about how to build and extend OpenSearch Dashboards - which APIs, plugins, resources, and services are available to speed development
+3. Discuss OpenSearch Dashboard roadmap and technical initiatives
+
+Signing up isn't required to attend - all OpenSearch Dashboards contributors or interested developers are welcome as participants.
+
+Bring your ideas and projects early, while you still have time and flexibility to make significant changes.
+
+### When
+
+Every other Thursday, 10AM-11AM PT.
+
+### How to sign up
```

Comment:
Nit: A link to the forum here would be useful. This can just be a link to the Dashboard channel there for now.

Reply:
Yeah, I'll add it once we have the forum posts. Some of this stuff is a little circular 😁